src/main/java/core/basesyntax/lotteryball/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.lotteryball;
+
+import core.basesyntax.enums.Colors;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method